Update 2023-12-06-tsc.md

asamborski opened the pull request:
Minutes from December 6, 2023 TSC meeting

asamborski committed:
Update 2023-12-06-tsc.md

Minutes from December 6, 2923 TSC meeting

jonnystoten commented on meeting-tsc/2023-12-06-tsc.md:
```diff
@@ -3,9 +3,51 @@
+
+The goal is to give the community a structured way to interact with the design of OPK rather than just the code. 
+
+For now, we think a Markdown file in GitHub makes sense (within the `openpubkey/openpubkey` repo) so folks can make PRs against it. We would like to adopt the RFC that defines usage of "must," "should," and "may." Diagrams in Markdown can use Mermaid or if a more complex diagram is necessary, can always embed an image. 
```
[RFC-2119](https://www.ietf.org/rfc/rfc2119.txt) is what I was thinking of!

asamborski replied:
Added this in! [debb349](https://github.com/openpubkey/community/pull/7/commits/debb34957cc01b09f022a0671100d1705480c41d)

jonnystoten commented on meeting-tsc/2023-12-06-tsc.md:
```diff
@@ -3,9 +3,51 @@
+
+The goal is to give the community a structured way to interact with the design of OPK rather than just the code. 
+
+For now, we think a Markdown file in GitHub makes sense (within the `openpubkey/openpubkey` repo) so folks can make PRs against it. We would like to adopt the RFC that defines usage of "must," "should," and "may." Diagrams in Markdown can use Mermaid or if a more complex diagram is necessary, can always embed an image. 
```
I mentioned https://github.com/in-toto/attestation/tree/main/spec/v1 and https://github.com/secure-systems-lab/dsse/blob/master/protocol.md as nice examples of specs in markdown on Github.

EthanHeilman replied:
DSSE Protocol looks really nice. It would be a nice example to follow

asamborski replied:
Added these in. [debb349](https://github.com/openpubkey/community/pull/7/commits/debb34957cc01b09f022a0671100d1705480c41d)

asamborski committed:
Update 2023-12-06-tsc.md

Update to include reference markdown specs and specific RFC mentioned in the meeting
